eox_nelp/user_authn/tests/test_views.py

```diff
@@ -0,0 +1,166 @@
+"""This file contains all the test for the Nelp registration form factory.
+Classes:
+    RegistrationFormFactoryTestCase: Test NelpRegistrationFormFactory login and class manipulation.
+"""
+from django.test import TestCase
+from django.utils.translation import gettext_lazy as _
+from mock import Mock
+
+from eox_nelp.edxapp_wrapper.site_configuration import configuration_helpers
+from eox_nelp.edxapp_wrapper.user_api import accounts
+from eox_nelp.user_authn.views.registration_form import NelpRegistrationFormFactory
+
+
+def mock_responses(responses):
+    """Function that manage the response of a mock based in its args.
+    You configre using a dict defining in the dict they key of each the args is used the response
```

Comment:
```suggestion
    You configure using a dict defining in the dict they key of each the args is used the response
```
However is not clear for me what you want to say